packages/atomic/src/components/common/atomic-rating/rating-utils.spec.ts

```diff
@@ -0,0 +1,43 @@
+import {describe, expect, it} from 'vitest';
+import {FieldValueIsNaNError} from '../../commerce/product-template-component-utils/error';
```

Comment:
nit; might be worth standardizing: I'd expect the same import format between the spec and the src file.